tests/driver_sx127x/main.c

```diff
@@ -269,6 +271,37 @@ int listen_cmd(int argc, char **argv)
     return 0;
 }
 
+int syncword_cmd(int argc, char **argv)
+{
+    if(argc < 2) {
```

Comment:
This file needs a round of uncrustify. There are spaces like this missing at some place. And there are extra spaces after cast types.

Reply:
ok, I will

Reply:
done!